packages/steppp-react/src/index.test.tsx

```diff
@@ -0,0 +1,185 @@
+import { render, screen } from "@testing-library/react";
+import { describe, it, expect } from "vitest";
+import { Provider, Step, useSteppp, Wrapper } from ".";
+
+describe("Using standard implementation.", () => {
```

Comment:
# 🎸 

Good work on the tests. I'd like to see boundary testing. For example:

- Moving backwards from the 1st step does nothing
- Moving forwards from the last step does nothing